samples/java/reference-app/pom.xml

```diff
@@ -10,12 +10,12 @@
   <url>http://maven.apache.org</url>
 
   <properties>
-    <java.version>1.8</java.version>
+    <java.version>17</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <appencryption.version>0.2.3</appencryption.version>
-    <junit.jupiter.version>5.9.3</junit.jupiter.version>
-    <junit.platform.version>1.9.3</junit.platform.version>
-    <jacoco.version>0.8.10</jacoco.version>
+    <appencryption.version>0.2.5</appencryption.version>
+    <junit.jupiter.version>5.9.2</junit.jupiter.version>
+    <junit.platform.version>1.9.2</junit.platform.version>
+    <jacoco.version>0.8.8</jacoco.version>
```

Comment:
Were these package downgrades intentional?

Reply:
No, I am stuck in a weird rebase conflict state. I'll fix these.